crates/bitwarden/src/crypto.rs

```diff
@@ -449,6 +493,32 @@ mod tests {
                 166, 209, 247, 194, 87, 232, 26, 48, 85, 193, 249, 179, 155
             ]
         );
+
+        let (key, mac) = stretch_key_password(
```

Comment:
I think we should add a new test scenario for these? Should assist with tracking down the failing test.

Reply:
During the test you get the stack trace indicating the failing line, which should help already, so I consider the separation more of a style choice, but if you prefer I can move it to a separate test function.

Reply:
You do, but you have to look at the source code to understand what failed, separate methods lets you immediately know the method name which to me adds some value.